jazzmin/templates/admin/edit_inline/stacked.html

```diff
@@ -6,28 +6,33 @@
             {{ inline_admin_formset.formset.non_form_errors }}
 
             {% for inline_admin_form in inline_admin_formset %}
-                <div class="panel inline-related{% if inline_admin_form.original or inline_admin_form.show_url %} has_original{% endif %}{% if forloop.last and inline_admin_formset.has_add_permission %} empty-form last-related{% endif %}" id="{{ inline_admin_formset.formset.prefix }}-{% if not forloop.last %}{{ forloop.counter0 }}{% else %}empty{% endif %}">
+                <div class="panel inline-related{% if inline_admin_form.original or inline_admin_form.show_url %} has_original{% endif %}{% if forloop.last and inline_admin_formset.has_add_permission %} empty-form last-related{% endif %}"
+                     id="{{ inline_admin_formset.formset.prefix }}-{% if not forloop.last %}{{ forloop.counter0 }}{% else %}empty{% endif %}">
+
                     <div class="card card-outline {% if not inline_admin_form.original %}new-stacked card-success{% else %}card-secondary{% endif %}">
                         <div class="card-header">
                             <h3 class="card-title">
+                                <span class="card-tools text-sm">
                                 {% if inline_admin_form.original %}
                                     {% if inline_admin_form.model_admin.show_change_link and inline_admin_form.model_admin.has_registered_model %}
-                                        <span class="card-tools">
                                             <a href="{% url inline_admin_form.model_admin.opts|admin_urlname:'change' inline_admin_form.original.pk|admin_urlquote %}" class="{% if inline_admin_formset.has_change_permission %}inlinechangelink{% else %}inlineviewlink{% endif %}">
                                             {% if inline_admin_formset.has_change_permission %}
-                                                <i class="fas fa-pencil-alt fa-sm float-right"></i>
+                                                <i class="fas fa-pencil-alt fa-sm"> </i>
                                             {% else %}
-                                                <i class="fas fa-eye fa-sm float-right"></i>
+                                                <i class="fas fa-eye fa-sm"> </i>
                                             {% endif %}
                                             </a>
-                                        </span>
+                                            {{ inline_admin_form.original }}
```

Comment:
This makes the title line disappear when `show_change_link = False`

Reply:
aah i had wondered why you moved it out